cumulusci/tasks/bulkdata/load.py

```diff
@@ -485,3 +522,128 @@ def _expand_mapping(self):
                         mapping["lookups"][l]["after"] = None
 
                     self.after_steps[after][name] = mapping
+
+    def _is_person_account_column_exists(self, mapping):
+        """
+        Returns if "IsPersonAccount" is a column in mapping's table.
+        """
+        return (
+            self.models[mapping.get("table")].__table__.columns.get("IsPersonAccount")
+            is not None
+        )
+
+    def _is_person_accounts_enabled(self, mapping):
+        """
+        Returns if the mapping supports person accounts meaning:
+        - The mapping has a "IsPersonAccount" column
+        - Person Accounts is enabled in the org.
+        Caches is_person_accounts_enabled response which consumes a describe
+        call.
+        """
+        if self._is_person_account_column_exists(mapping):
+            if self._person_accounts_enabled is None:
+                self._person_accounts_enabled = is_person_accounts_enabled(self)
+            return self._person_accounts_enabled
+        return False
+
+    def _update_person_account_name_as_blank(self, mapping) -> None:
+        """
+        If a "Name" Salesforce Field is mapped to a column, updates mapping's
+        table records where IsPersonAccount is "true" setting "Name" field as
+        blank.
+        """
+        # Check if Account.Name is in mapping
+        fields = mapping.get("fields", {})
+        for api_name, column_name in fields.items():
+            if api_name.lower() == "name":
+                # Update table
+                table = self.models[mapping["table"]].__table__
+                self.session.connection().execute(
+                    table.update()
+                    .where(table.columns.get("IsPersonAccount") == "true")
+                    .values(**{column_name: ""})
+                )
+                self.session.flush()
+                return
+
+    def _filter_out_person_account_records(self, query, model):
```

Comment:
This is its own method so it's easy to test when this is called in `_query_db` method.